### ISSUE TITLE
Pygil

### ISSUE THREAD

**Issue by huwdp** [lightspark/lightspark#779]
I think the implemented percentage should be higher on the Lightspark homepage.

So for example Pygil says that ```flash.ui.ContextMenu.isSupported (getter)``` is missing but it's in the codebase here (https://github.com/lightspark/lightspark/blob/48506cbc861ed138d6b835407cd0e408c293da79/src/scripting/flash/ui/ContextMenu.cpp#L33) as a constant. There are quite a few instances of these.

---

Edit, after doing some research, I can only find 16 of these so I doubt it would push the implementation percentage that much higher.

**Comment by dbluelle** [comment]
This one was implemented wrong: "isSupported" is a property, not a constant. I've fixed that.
Can you post the list of the other ones you found?

About the implementation percentage:
This is the percentage for all methods (that includes those that are only available in Adobe AIR), so the percentage for the flash player should be higher. Maybe someone with skills in Python can make a script that excludes the AIR methods.

**Comment by huwdp** [comment]
I think the ones below _might_ be fixable. I also included pygil output to this ticket. [log.zip](https://github.com/lightspark/lightspark/files/6744241/log.zip)

**Constants:**
- flash.concurrent.Condition.isSupported (getter)
- flash.accessibility.Accessibility.active (getter)

**Missing setter:**
- flash.display.Stage.height (setter)
- flash.display.Stage.width (setter)

I'll try to go through more of these types of things in the near future. I stopped coding on Lightspark as I can't compile AS3 files if they contain APIs, e.g. URLRequestDefaults won't compile in FlashDevelop with the Flash (Flex I think) compiler.

---

I'll have a look at removing AIR stuff from pygil, should definitely be possible. Flash Matrix [here](https://huwdp.github.io/Flash-Matrix/) (https://huwdp.github.io/Flash-Matrix/) includes AS3 stuff but not AIR.
